pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -104,17 +112,20 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 
 	// one of the items in this list should be the project.  We are going to locate it, remove it from the list, create it separately
 	var projectFromTemplate *projectapi.Project
+	var lastRoleBinding *authorizationapi.RoleBinding
 	objectsToCreate := &kapi.List{}
 	for i := range list.Objects {
 		if templateProject, ok := list.Objects[i].(*projectapi.Project); ok {
 			projectFromTemplate = templateProject
-
-			if len(list.Objects) > (i + 1) {
-				objectsToCreate.Items = append(objectsToCreate.Items, list.Objects[i+1:]...)
-			}
+			// don't add this to the list to create.  We'll create the project separately.
 			break
 		}
 
+		if roleBinding, ok := list.Objects[i].(*authorizationapi.RoleBinding); ok {
+			// keep track of the rolebinding, but still add it to the list
+			lastRoleBinding = roleBinding
```

Comment:
> will this bite us if there are multiple role bindings in the template?

Not in sane (or even semi-sane) etcd conditions.  The last rolebinding created will be the last in the log and the last observed.